src/app/content/constants.ts

```diff
@@ -16,3 +17,10 @@ export const highlightStyles: Array<{label: HighlightColorEnum, passive: string,
   {label: HighlightColorEnum.Purple, passive: '#cbcfff', focused: '#545ec8'},
   {label: HighlightColorEnum.Pink, passive: '#ffc5e1', focused: '#de017e'},
 ];
+
+export const practiceQuestionStyles: PracticeQuestionStyles[] = [
+  {label: 'incorrect', passive: '#C22032', focused: '#C22032', fontColor: '#FFFFFF', hovered: '#C22032'},
+  {label: 'correct', passive: '#77AF42', focused: '#77AF42', fontColor: '#FFFFFF', hovered: '#77AF42'},
```

Comment:
where do these colors come from? are they not from the theme palette?

Reply:
not all of them... fixed the ones that are.